src/main/java/com/smartbear/readyapi/client/result/TestStepResult.java

```diff
@@ -50,6 +50,15 @@ public Long getTimeTaken() {
         return testStepResultReport.getAssertionStatus();
     }
 
+    public TestStepResultReport.AssertionStatusEnum getStatusForAssertion(String assertionName) {
+        for (String message : testStepResultReport.getMessages()) {
+            if (message.startsWith("[" + assertionName + "]")) {
```

Comment:
I realized that there is no way to get the list of assertions from TestStepResult in TestServer. So, we will live with this solution for the time being.